webmessaging/Channel_postMessage_Blob.htm

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title> postMessage() with a Blob </title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id=log></div>
+<script>
+    var TARGET = null;
+    var SOURCE = null;
+    var description = "Messages can contain Blobs.";
+
+    var t = async_test("Test Description: " + description);
+
+    var channel = new MessageChannel();
+    SOURCE = channel.port1;
+    TARGET = channel.port2;
+    TARGET.start();
+    TARGET.addEventListener("message", t.step_func(TestMessageEvent), true);
+
+    (function() {
+      SOURCE.postMessage({blob: new Blob(['foo', 'bar'])});
+    })();
+    if (self.gc) self.gc();
```

Comment:
@foolip Is there any guidance on tests making use of nonstandard APIs in ways that won't break other browsers but will make the tests less meaningful in them?

Reply:
I'm afraid not in general beyond "don't do that". There is something similar in https://github.com/w3c/web-platform-tests/blame/master/streams/resources/test-utils.js added by https://github.com/w3c/web-platform-tests/pull/2489.

It's tempting to do something with testharnessreport.js around GC like that, but a problem is that with Oilpan in Chromium, a full GC isn't actually possible at any point, so we also have tests that do asyncGC:
https://cs.chromium.org/chromium/src/third_party/WebKit/LayoutTests/resources/js-test.js?l=744

Still, maybe we could figure something out. Can you file an issue with the infra label? @jgraham FYI.

Here, I think keeping this around is fine if it's documented with a TODO linking to an issue, it won't do any harm.

Reply:
(If it were prefixed APIs being used I'd see things differently though, I recently removed the mechanisms for that.)

Reply:
Filed https://github.com/w3c/web-platform-tests/issues/7899.

Reply:
> Here, I think keeping this around is fine if it's documented with a TODO linking to an issue, it won't do any harm.

Does that mean you want me to add a comment with a link to that issue to this CL?

Reply:
Yes.